src/EduManagementLab.Api/Controllers/CoursesController.cs

```diff
@@ -149,6 +149,21 @@ public ActionResult<CourseDto> AddCourseMembership(Guid courseId, Guid userId, D
 
         }
 
+        [HttpPut]
+        [Route("{courseId}/UpdateMembership/{userId}")]
```

Comment:
Rename route to "{courseId}/Memberships/{userId}"
The PUT method tells that the resource should be replaced/updated.

Reply:
I rename it to "{courseId}/Memberships/{userId}"